### PR TITLE
feat(ARC-2159): add description to collection endpoints

### PR DESCRIPTION
**Pull request by RobbeBierebeeck**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "start": "nest start",
     "__predev": "npm run generate-database-types",
     "prestart:debug": "npm run generate-database-types",
-    "dev": "dotenv -e .env/.env.local nest start --watch",
```
Comment (bertyhell) [inline package.json:21]: this didn't fix your debug issue, so lets remove this change so other devs do not get confused by the server not restarting anymore

File: src/modules/collections/controllers/collections.controller.ts [hunk 2]
```diff
@@ -160,6 +160,7 @@ export class CollectionsController {
 			{
 				name: createCollectionDto.name,
 				user_profile_id: user.getId(),
+				description: createCollectionDto.description,
```
Comment (bertyhell) [inline src/modules/collections/controllers/collections.controller.ts:163]: move this line up one line

File: src/modules/collections/dto/collections.dto.ts [hunk 3]
```diff
@@ -12,6 +12,15 @@ export class CreateOrUpdateCollectionDto {
 		example: 'Favorites',
 	})
 	name: string;
+
+	@IsString()
+	@IsOptional()
+	@ApiPropertyOptional({
+		type: string,
+		description: 'The description of the collection',
```
Comment (bertyhell) [inline src/modules/collections/dto/collections.dto.ts:20]: we're migrating all "collection" names to "folder" when you're touching the code or creating new code
```suggestion
		description: 'The description of the folder',
```

**Review by bertyhell — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (RobbeBierebeeck) [reply package.json:21]: This --watch flag did never work

**Pull request merged**